hoomd/md/constrain.py

```diff
@@ -209,7 +209,7 @@ class Rigid(Constraint):
         - ``diameters`` (list[float]): List of diameters of constituent
           particles
 
-        Type: `TypeParameter[``particle_type``, `dict`]
+        Type: `TypeParameter` [``particle_type``, `dict`]
```

Comment:
The `TypeParameter` type definitions are inconsistent throughout the code, I did my best to improve whatever was already there based on examples elsewhere in the code.

Some use a construction like this, and I'm not sure if we have a reason to prefer one over the other:

```
    Attributes:
        params (`TypeParameter` [``dihedral type``, `dict`]):
            The parameter of the harmonic bonds for each dihedral type. The
            dictionary has the following keys:
```

Reply:
I have used the Sphinx `.. py:attribute::` syntax in places where Google formatting for the attribute name and type is longer than the line length limit. As discussed in #1001 I do not accept the solution that applies noqa to all lines in a docstring as that to easily allows for many style violations beyond the local line where Google style demands it.